Containers/docker-socket-proxy/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:latest
-FROM haproxy:2.9.7-alpine3.19
+FROM haproxy:2.9.7-alpine3.20
```

Comment:
```suggestion
FROM haproxy:3.0.2-alpine3.20
```

Reply:
lets do that in a different pr

Reply:
ok, not sure why there was no automatic PR for this

Reply:
I think because there is no 3.0.2-alpine3.19 available...

Reply:
https://github.com/nextcloud/all-in-one/pull/4860